config/locales/en/casework.yml

```diff
@@ -118,7 +118,7 @@ en:
     overall_offence_class: Overall offence class
     overview: Overview
     partner: Partner
-    post_submission_evidence: Post submission evidence
+    post_submission_evidence: PSE
```

Comment:
Are we happy / have we checked that they are all good with 'PSE'?

Reply:
Nope haven't checked with anyone. Do you mean ucd or users or both?

Reply:
Ultimately users, but UCD might have a steer for now. Not a big deal - I just wanted to ask to make sure it's a term everyone is going to understand. As a general rule we should be steering away from acronyms where possible